.gitignore

```diff
@@ -31,6 +31,7 @@ build/Release
 
 # Dependency directories
 node_modules/
+!packages/opentelemetry-tracer/test/instrumentation/node_modules
```

Comment:
Should this be in the gitignore of that package?

Reply:
Makes sense, done.